der/derive/src/attributes.rs

```diff
@@ -331,7 +331,7 @@ impl AttrNameValue {
                 self.value
                     .value()
                     .parse()
-                    .unwrap_or_else(|_| abort!(self.name, "error parsing `{}` attribute")),
+                    .unwrap_or_else(|_| abort!(self.name, "error parsing `{}` attribute", name)),
```

Comment:
`self.name` and `name` are the same thing here. Maybe just change it to this?
```suggestion
                    .unwrap_or_else(|_| abort!(self.name, "error parsing attribute")),
```

Reply:
Done.